src/rhub/api/utils.py

```diff
@@ -64,3 +65,7 @@ def condition_eval(expr, params):
     elif expr[0] == 'param_in':
         return expr[1] in params and expr[2] in params[expr[1]]
     raise ValueError(f'Unknown operation {expr[0]!r}')
+
+
+def choose_from(number):
```

Comment:
[`random.randrange`](https://docs.python.org/3/library/random.html?highlight=random#random.randrange)

Reply:
@fpob Fixed in the last commit.